Geometry/Records/src/ES_MuonDDDConstants.cc

```diff
@@ -0,0 +1,4 @@
+#include "Geometry/MuonNumbering/interface/MuonDDDConstants.h"
```

Comment:
@cmsbuild - it is very strange that this file is not in IB. Any idea what's wrong?